pkg/cmd/server/kubernetes/network/network.go

```diff
@@ -194,6 +200,26 @@ func (c *NetworkConfig) RunProxy() {
 	endpointsConfig.RegisterEventHandler(endpointsHandler)
 	go endpointsConfig.Run(utilwait.NeverStop)
 
+	// Start up healthz server
+	if len(c.ProxyConfig.HealthzBindAddress) > 0 {
+		healthzServer.Run()
+	}
+
+	// Start up a metrics server if requested
+	if len(c.ProxyConfig.MetricsBindAddress) > 0 {
+		mux := http.NewServeMux()
+		mux.HandleFunc("/proxyMode", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
name it alpha or something?  this is like config-lite.

Reply:
This is consistent with upstream, so we can't rename it.